### DISCUSSION TITLE
How do I select the language of the movie ?

### DISCUSSION THREAD

**Discussion by Philetjosie** [NickvisionApps/Parabolic#473] [Q&A] [answered]
Hello, many thanks for the developper to have added the ability to select the movie audio language, with the latest release of Parabolic.
However, I have not found how to do it. I tried with different Arte.tv movies (for exemple this one: https://www.arte.tv/fr/videos/062208-000-A/ete-violent/  where you can see there is options to get it in french or in original version (italian), but I have not found where in the interface of parabolic I can do the choice ?
Many thanks and best regards.

**Comment by nlogozzo** [comment, marked as answer]
Hi! If yt-dlp detects multiple audio languages for a download, it will display a list of selectable languages in the `Add Download` dialog. 

So for this video, if you go to add a new download, paste the link and validate it you should then see this:
![image](https://github.com/NickvisionApps/Parabolic/assets/17648453/9bf9e2d3-8194-491a-9244-40e06e85e897)
and you can select the language from there.

If you don't see this option, please go to `About Parabolic` in the main menu, select `Troubleshooting`, then `Debugging Information`, and copy the text and paste it here.

**Reply by Philetjosie** [reply]
Thanks for your reply. In fact, this menu does not appears. Here are the debugging informations:

org.nickvision.tubeconverter
2023.7.3-beta1
GTK 4.10.4
libadwaita 1.3.3
Snap
yt-dlp 2023.03.04
psutil 5.9.5
ffmpeg version n6.0-26-g3f345ebf21
aria2 version 1.36.0
fr-FR
LANG=fr_FR.UTF-8
LANGUAGE=
LC_CTYPE="fr_FR.UTF-8"
LC_NUMERIC="fr_FR.UTF-8"
LC_TIME="fr_FR.UTF-8"
LC_COLLATE="fr_FR.UTF-8"
LC_MONETARY="fr_FR.UTF-8"
LC_MESSAGES="fr_FR.UTF-8"
LC_PAPER="fr_FR.UTF-8"
LC_NAME="fr_FR.UTF-8"
LC_ADDRESS="fr_FR.UTF-8"
LC_TELEPHONE="fr_FR.UTF-8"
LC_MEASUREMENT="fr_FR.UTF-8"
LC_IDENTIFICATION="fr_FR.UTF-8"
LC_ALL=

**Reply by nlogozzo** [reply]
Could you update the snap and try again?
CC @soumyaDghosh it works fine on the flatpak version and running locally...

**Reply by Philetjosie** [reply]
Hello, I did "snap refresh", and now it works !!! Great, and many thanks !
Here is the debuging information now:

org.nickvision.tubeconverter
2023.7.3-rc1
GTK 4.10.4
libadwaita 1.3.3
Snap
yt-dlp 2023.07.06
psutil 5.9.5
ffmpeg version n6.0-26-g3f345ebf21
aria2 version 1.36.0
fr-FR
LANG=fr_FR.UTF-8
LANGUAGE=
LC_CTYPE="fr_FR.UTF-8"
LC_NUMERIC="fr_FR.UTF-8"
LC_TIME="fr_FR.UTF-8"
LC_COLLATE="fr_FR.UTF-8"
LC_MONETARY="fr_FR.UTF-8"
LC_MESSAGES="fr_FR.UTF-8"
LC_PAPER="fr_FR.UTF-8"
LC_NAME="fr_FR.UTF-8"
LC_ADDRESS="fr_FR.UTF-8"
LC_TELEPHONE="fr_FR.UTF-8"
LC_MEASUREMENT="fr_FR.UTF-8"
LC_IDENTIFICATION="fr_FR.UTF-8"
LC_ALL=

**Reply by soumyaDghosh** [reply]
That's why snaps often auto refreshes, after some days

**Reply by soumyaDghosh** [reply]
BTW, @Philetjosie can you kindly share the video link, as I would like to test why 2023.7.3-beta1 didn't work!

**Reply by Philetjosie** [reply]
Yes, it was in my first message: https://www.arte.tv/fr/videos/062208-000-A/ete-violent/ 
I now can choose between french, italian and german.
Regards.

**Reply by nlogozzo** [reply]
> Hello, I did "snap refresh", and now it works !!! Great, and many thanks !

Glad that fixed it :)
